### ISSUE TITLE
POS-Tags for german Nouns when written in lower case

### ISSUE THREAD

**Issue by ctrado18** [explosion/spaCy#2546]
Hi,

when a noun is written in lower case like `tisch` spaCy does not tag the proper Tag `Noun`. But for chatbots you write typically everything small. Is it possible to handle this in spaCy somehow?

**Comment by ines** [comment]
The German model was trained on the TIGER corpus, which mostly consists of general-purpose news text (that's also why it's called `de_core_news`). News texts generally uses proper capitalisation, and the model is sensitive to spelling, since it can hold important clues for its predictions.

There's currently an open enhancement proposal in #1922 that discusses a German model for lowercase text (e.g. by re-training the same model on the same data and labels, but lowercasing the text before training.) Making this work will require some experimentation, though.

If you're working with chat text, you'll likely also want to post-train the model with more examples specific to your data and domain. For example, besides the capitalisation, chat texts also tend to use punctuation differently, and often include incomplete sentences and more imperatives ("bestell mir ein taxi" etc., which is pretty rare in general news and web text.) So you'll almost *always* want to update a pre-trained model with more examples and fine-tune it for your use case. See [the training docs](https://spacy.io/usage/training) for  examples on how to update existing models.
